docker/postgres/docker-compose.yml

```diff
@@ -71,6 +71,29 @@ services:
         # service_started, not service_healthy as pinot and deps can take longer than 60s to start
         condition: service_started
 
+  hypertrace-alert-engine:
```

Comment:
@kotharironak , I see what you have done here. Ideally the way we had 2 different files earlier makes more sense considering I don't want alerting when I am trying it out for first time anyways because I am just trying out platform. 

If directory or command is the problem, we can have duplicate files but let's have it in 2 steps. First step will be `quick-start with Hypertrace` and then `try out alerting with Hypertrace`. It will be cleaner way because you will need to stop and start the whole stack again in your case anyways but then there's additional thing to do that is comment these lines etc. 

cc: @rish691

Reply:
@JBAhire In that case, we need a copy of the same files in postgres folder. I can do that as well. I will copy alerting files in postgres folder then.

Reply:
As discussed offline with @JBAhire, let's keep the default + postgress folder in sync. And, we will go with one simple command.
`docker-compose -f <default file> -f <alerting file>`